src/Illuminate/Collections/helpers.php

```diff
@@ -15,6 +15,10 @@
      */
     function collect($value = [])
     {
+        if ($value instanceof \Illuminate\Database\Eloquent\Model) {
+            $value = [$value];
+        }
+
         return new Collection($value);
```

Comment:
@cosmastech You can do,

```suggestion
        return Collection::wrap($value);
```